Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,511 @@
+        {
+            m_tabWidget->setCurrentIndex(1);
+            m_nextButton->setText(tr("Next"));
+        }
+        else if (m_tabWidget->currentIndex() == 1)
+        {
+            m_backButton->setVisible(false);
+            m_tabWidget->setCurrentIndex(0);
+        }
+        else if (m_tabWidget->currentIndex() == 0)
+        {
+            m_backButton->setVisible(false);
+        }
+    }
+
+    void CreateAGemScreen::HandleNextButton()
+    {
+        if (m_tabWidget->currentIndex() == 0)
+        {
+            bool templateLocationIsValid = ValidateGemTemplateLocation();
+            if (!m_formFolderRadioButton->isChecked())
+            {
+                m_backButton->setVisible(true);
+                m_tabWidget->setCurrentIndex(1);
+            }
+            else if (templateLocationIsValid && m_formFolderRadioButton->isChecked())
+            {
+                m_backButton->setVisible(true);
+                m_tabWidget->setCurrentIndex(1);
+            }
+        }
+        else if (m_tabWidget->currentIndex() == 1)
+        {
+            bool displayNameIsValid = ValidateGemDisplayName();
+            bool systemNameIsValid = ValidateGemSystemName();
+            bool licenseNameIsValid = ValidateLicenseName();
+            bool globalGemTagIsValid = ValidateGlobalGemTag();
+            bool optionalGemTagsAreValid = ValidateOptionalGemTags();
+            if (systemNameIsValid && licenseNameIsValid && globalGemTagIsValid && displayNameIsValid && optionalGemTagsAreValid)
+            {
+                m_createAGemInfo.m_displayName = m_gemDisplayName->lineEdit()->text();
+                m_createAGemInfo.m_name = m_gemSystemName->lineEdit()->text();
+                m_createAGemInfo.m_summary = m_gemSummary->lineEdit()->text();
+                m_createAGemInfo.m_requirement = m_requirements->lineEdit()->text();
+                m_createAGemInfo.m_licenseText = m_license->lineEdit()->text();
+                m_createAGemInfo.m_licenseLink = m_licenseURL->lineEdit()->text();
+                m_createAGemInfo.m_licenseText = m_license->lineEdit()->text();
+                m_createAGemInfo.m_documentationLink = m_documentationURL->lineEdit()->text();
+                m_createAGemInfo.m_path = m_gemLocation->lineEdit()->text();
+
+                m_tabWidget->setCurrentIndex(2);
+                m_nextButton->setText(tr("Create"));
+
+            }
+        }
+        else if (m_tabWidget->currentIndex() == 2)
+        {
+            bool creatorNameIsValid = ValidateCreatorName();
+            bool repoURLIsValid = ValidateRepositoryURL();
+            if (creatorNameIsValid && repoURLIsValid)
+            {
+                m_createAGemInfo.m_creator = m_creatorName->lineEdit()->text();
+                m_createAGemInfo.m_repoUri = m_repositoryURL->lineEdit()->text();
+                auto result = PythonBindingsInterface::Get()->CreateGem(m_gemTemplateLocation->lineEdit()->text(), m_createAGemInfo);
```

Comment:
we need to handle this result and display something to the user on success/failure.  In my opinion we should not just close the wizard and return to the catalog without providing feedback
```suggestion
                auto result = PythonBindingsInterface::Get()->CreateGem(m_gemTemplateLocation->lineEdit()->text(), m_createAGemInfo);
```